### DISCUSSION TITLE
Is it possible to implement third color mode in Chakra UI?

### DISCUSSION THREAD

**Discussion by matt765** [chakra-ui/chakra-ui#7596] [Q&A]
I would like to have 3 color modes instead of just dark and light. Does anyone know how to do this or have seen an example of implementation?

**Comment by keithburgie** [comment]
Curious about this as well, to add something like a "high-contrast" or "large-text" mode.

**Comment by matt765** [comment]
I created additional themes with Chakra UI, but it's more like a workaround instead of actual solution because I have 2 sets of dark/light colors that I switch with JavaScript.

https://github.com/matt765/daydash/tree/master/src/theme
